src/Storage/Storage.Management/StorageAccount/RestoreAzStorageBlobRange.cs

```diff
@@ -12,15 +12,17 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using Track2 = Azure.ResourceManager.Storage;
+using Track2Models = Azure.ResourceManager.Storage.Models;
```

Comment:
Should update Storage.Management.csproj, by remove following line to include the file in az.storage.

`<Compile Remove="StorageAccount\RestoreAzStorageBlobRange.cs" />`

Reply:
Done.